src/data/services/LmsApiService.js

```diff
@@ -324,6 +326,20 @@ class LmsApiService {
       formData,
     );
   }
+
+  static getUserPreferences(username) {
```

Comment:
Is `getUserPreferences` and `patchUserPreferences` still used in this implementation? I think this is left over from before the decision to use a cookie instead of the DB user preferences?